libqtile/widget/graph.py

```diff
@@ -200,7 +200,10 @@ def __init__(self, **config):
         self.oldvalues = self._getvalues()
 
     def _getvalues(self):
-        with open('/proc/stat') as file:
+        proc = '/proc/stat'
+        if platform.system() == 'FreeBSD':
```

Comment:
You forget to import platform on the top. `import platform`.  This was the reason that the test fails.